### PR TITLE
Add regexp module and tests

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY
Add regexp module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

community.zabbix.zabbix_regexp

##### ADDITIONAL INFORMATION
Add [regexp](https://www.zabbix.com/documentation/6.0/en/manual/api/reference/regexp/object) module and tests.

### CODE REVIEW

File: plugins/modules/zabbix_regexp.py [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+        try:
+            parameter = [regexp['regexpid']]
+            if self._module.check_mode:
+                self._module.exit_json(changed=True)
+            self._zapi.regexp.delete(parameter)
+            self._module.exit_json(changed=True, msg="Successfully deleted regular expression setting.")
+        except Exception as e:
+            self._module.fail_json(msg="Failed to delete regular expression setting: %s" % e)
+
+    def _compare_expressions(self, current_expressions, future_expressions):
```
Comment (D3DeFi) [inline plugins/modules/zabbix_regexp.py:160]: I wonder if https://github.com/ansible-collections/community.zabbix/blob/a3df0f1a9704085d232980491a29229df2d80e90/plugins/module_utils/helpers.py#L113 or https://github.com/ansible-collections/community.zabbix/blob/a3df0f1a9704085d232980491a29229df2d80e90/plugins/module_utils/helpers.py#L147 wouldn't fit here. What do you think?

**Review by D3DeFi — CHANGES_REQUESTED** [review]
Thank you for your PR. Please see my comment in the review

File: plugins/modules/zabbix_regexp.py (hunk 1 above)
Comment (masa-orca) [reply plugins/modules/zabbix_regexp.py]: @D3DeFi 
I apologize for taking much times.
Your suggestion is great to me! I use the function.
In addition, I have modified documents. Will you check commits?
By the way, which version will add this module? I need to change `version_added`.

**Comment by BGmot** [comment]
Tthanks @masa-orca! sorry for delay in merging, @D3DeFi please merge if you don't have other comments? I approved.

**Review by BGmot — APPROVED** [review]

**Review by pyrodie18 — APPROVED** [review]

**Pull request merged**